python/dolfinx/geometry.py

```diff
@@ -270,3 +270,32 @@ def compute_distance_gjk(
 
     """
     return _cpp.geometry.compute_distance_gjk(p, q)
+
+def determine_point_ownership(
+    mesh: Mesh,
+    points: npt.NDArray[np.floating], 
+    cells: typing.Optional[np.ndarray] = None,
+    padding: float = 0.0
```

Comment:
The C++ method does not have a default, make consistent.

Reply:
Added default padding = 0.0 to c++ method